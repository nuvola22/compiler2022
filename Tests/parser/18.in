function a(): Integer;
begin a := 10; end;

function b(): Integer;
begin b := 20; end;

begin
    writeln(a() + 30, b() + 40);
end.