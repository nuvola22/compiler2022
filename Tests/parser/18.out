      program
         a
            parameters:
            declarations:
            type:
               Integer
            begin
               :=
                  a
                  10
         b
            parameters:
            declarations:
            type:
               Integer
            begin
               :=
                  b
                  20
         begin
            call
               writeln
               +
                  call
                     a
                  30
               +
                  call
                     b
                  40
